### ISSUE TITLE
 Error using Import-Scan with Spotbugs

### ISSUE THREAD

**Issue by viniciusnavarro** [DefectDojo/django-DefectDojo#5065]
**Bug description**
When importing (using import-scan) from a SpotBugs .xml file, I get a 500 error: `("message" error: "Internal server error, check logs for details"),`
when looking deeper into one of the logs I found this:

> [08/Sep/2021 19:30:36] ERROR [dojo.importers.utils:27] int() argument must be a string, a bytes-like object or a number, not 'NoneType'
> [08/Sep/2021 19:30:36] WARNING [dojo.importers.utils:28] Error in parser: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
> [08/Sep/2021 19:30:36] ERROR [django.request:224] Internal Server Error: /api/v2/import-scan/

The strangest thing is that in version 1.12.1 I import the same file without any problem.

**Steps to reproduce**
1. Go to Import Scan Results
2. Select SpotBugs Scan
3. Choose the report file in our case Spotbugs
4. See error:   "An exception error occurred during the report import:int() argument must be a string, a bytes-like object or a number, not 'NoneType'"

Using the API the same exception happens.

**Expected behavior**
Import SpotBugs file to DD without problems

**Deployment method** *(select with an `X`)*
- [X] Docker
- [ ] Kubernetes
- [ ] GoDojo
- [ ] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: Ubuntu 20.04
 - DefectDojo Commit Message: [2021-08-31 12:37:55 -0500] 00b425742: Merge pull request #5014 from DefectDojo/release/2.2.0 [ (HEAD -> master, tag: 2.2.0, origin/master, origin/HEAD)]

**Comment by valentijnscholten** [comment]
The import code was refactored at some point between 1.12.1 and 2.0.0. Can you provide an example to reproduce this? The reports used by the unit tests are running fine.

**Comment by damiencarol** [comment]
I take this one.

**Comment by damiencarol** [comment]
More complete stack:
```
   findings = parser.get_findings(testfile, Test())
  File "/home/damien/dd2/dojo/tools/spotbugs/parser.py", line 91, in get_findings
    title = bug.find('ShortMessage').text
AttributeError: 'NoneType' object has no attribute 'text'
```

**Comment by damiencarol** [comment]
The bug come from a big difference in reports from `4.4.0` generated with Maven and the reports we have in our unittests.

**Comment by viniciusnavarro** [comment]
@valentijnscholten @damiencarol Here is the Spotbugs file I am trying to import, some data has been masked but the file is intact.
[spotbugsXml.txt](https://github.com/DefectDojo/django-DefectDojo/files/7137086/spotbugsXml.txt)

I renamed it to txt to be able to send.

**Comment by valentijnscholten** [comment]
will de fixed by #5068

**Comment by viniciusnavarro** [comment]
Very nice @valentijnscholten  

Thank you @damiencarol   and @valentijnscholten  for the help and attention.
